db/migrate/20200302123440_add_min_available_count_to_deployments.rb

```diff
@@ -0,0 +1,5 @@
+class AddMinAvailableCountToDeployments < ActiveRecord::Migration[5.2]
+  def change
+    add_column :deployments, :min_available_count, :integer, null: false, default: 1
```

Comment:
related to previous comment, this can be `min_available_replica`

Reply:
Noted.

Reply:
We have changed this in another migration.